Traceback (most recent call last):
  File "server_rl_predict.py", line 48, in <module>
    httpd = make_server('tensorflow-gpu-0.tensorflow-gpu.default.svc.cluster.local', 8002, application)
  File "/usr/lib/python3.5/wsgiref/simple_server.py", line 160, in make_server
    server = server_class((host, port), handler_class)
  File "/usr/lib/python3.5/socketserver.py", line 440, in __init__
    self.server_bind()
  File "/usr/lib/python3.5/wsgiref/simple_server.py", line 51, in server_bind
    HTTPServer.server_bind(self)
  File "/usr/lib/python3.5/http/server.py", line 138, in server_bind
    socketserver.TCPServer.server_bind(self)
  File "/usr/lib/python3.5/socketserver.py", line 454, in server_bind
    self.socket.bind(self.server_address)
OSError: [Errno 98] Address already in use
